lib/state/simperium/middleware.ts

```diff
@@ -269,6 +271,27 @@ export const initSimperium = (
             }
           }, 250);
         }
+        return result;
+      }
+
+      case 'REVISIONS_TOGGLE': {
+        const state = getState();
+        const showRevisions = state.ui.showRevisions;
+        const noteId = state.ui.openedNote;
+
+        if (noteId && showRevisions) {
+          setTimeout(() => {
```

Comment:
we may not need this delay here since we know we're opening the revisions slider. it's probably not bad to keep in in case someone opens and immediately closes it - we wouldn't need to issue the request. if that's the case though we'd want to check `showRevisions` from inside the `setTimeout` callback

probably the most straightforward thing here would be to remove the timeout and issue the fetch every time we open up the slider.